sandbox/public/onBeforeEventSentHelper.js

```diff
@@ -0,0 +1,40 @@
+// code to collect tnta and trigger the Analytics Hit
+const trackingServer = "ujsl.sc.omtrdc.net";
+const reportSuite = "ujslecommerce";
+
+function triggerAnalyticsHit({ analyticsPayload, visitorID }) {
```

Comment:
Is this duplicated in DataInsertionAPI.js or did you forget to delete it after breaking it out?

Reply:
yes, it is duplicated from DataInsertionApi. I can't import it here because it is outside the modules.